InvalidOperationException: No entry methods in dotnet/runtime build

See https://dev.azure.com/dnceng/public/_build/results?buildId=474524&view=logs&jobId=214ce225-49cc-5240-836e-21c9eea7fd5c&j=de7d46dd-46cc-5631-ddd7-a3c134a1a855&t=886708a9-f90e-5e10-9587-aa9f4cd21d9b from dotnet/runtime#1088

```
  Fatal error in IL Linker
  Unhandled exception. System.InvalidOperationException: No entry methods
     at Mono.Linker.Steps.MarkStep.Process()
     at Mono.Linker.Steps.MarkStep.Process(LinkContext context)
     at Mono.Linker.Pipeline.ProcessStep(LinkContext context, IStep step)
     at Mono.Linker.Pipeline.Process(LinkContext context)
     at Mono.Linker.Driver.Run(ILogger customLogger)
     at Mono.Linker.Driver.Execute(String[] args, ILogger customLogger)
     at Mono.Linker.Driver.Main(String[] args)
/Users/runner/runners/2.163.1/work/1/s/eng/illink.targets(132,5): error MSB6006: "dotnet" exited with code 134. [/Users/runner/runners/2.163.1/work/1/s/src/libraries/shims/generated/netstandard.csproj]
##[error]eng/illink.targets(132,5): error MSB6006: "dotnet" exited with code 134.
```

/cc @marek-safar

--- a/src/linker/Linker.Steps/MarkStep.cs
+++ b/src/linker/Linker.Steps/MarkStep.cs
@@ -204,8 +204,11 @@ namespace Mono.Linker.Steps {
 
 		void Process ()
 		{
+			//
+			// This can happen when linker is called on facade with all references skipped
+			//
 			if (QueueIsEmpty ())
-				throw new InvalidOperationException ("No entry methods");
+				return;
 
 			while (ProcessPrimaryQueue () || ProcessLazyAttributes () || ProcessLateMarkedAttributes ())
 

